dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/tckRunner/tck/tck-suite-lite.xml

```diff
@@ -0,0 +1,23 @@
+<!--Copyright (c) 2022 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="jakarta-concurrency" verbose="2" configfailurepolicy="continue">
+	<test name="jakarta-concurrency.tck.lite">
+		<packages>
+    		<package name="ee.jakarta.concurrent.api.*">
```

Comment:
Is this the correct package name (4 more instances below)?  I thought we switched to ee.jakarta.tck.concurrent?